programming_guide/mini_tutorial/aie2p.py

```diff
@@ -0,0 +1,124 @@
+# aie2p.py -*- Python -*-
+#
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# (c) Copyright 2025 Advanced Micro Devices, Inc. or its affiliates
+
+import numpy as np
+import sys
+
+from aie.iron import Program, Runtime, Worker, ObjectFifo
+from aie.iron.placers import SequentialPlacer
+from aie.iron.device import NPU2
+from aie.iron.controlflow import range_
+
+import aie.iron as iron
+
+# The device on which the Program will be executed. Should match available hardware architecture.
+# You can see a list of available devices in python/iron/device/__init__.py
+dev = NPU2()
+
+# Define tensor types
+# These represent the size and datatype of the data tensors on which we do compute or move
+# with the ObjectFifo primitive.
+num_elements = 48
+data_type = np.int32
+tile_ty = np.ndarray[(num_elements,), np.dtype[data_type]]
+
+
+# JIT decoratior for IRON
+# Decorator to compile an IRON kernel into a binary to run on the NPU.
+@iron.jit(is_placed=False)
+def aie2p():
+    # Dataflow with ObjectFifos
+    # ObjectFifos represent a dataflow connection between endpoints in the AIE array.
+    # The IRON placement step relies on ObjectFifoHandles to infer the endpoints of ObjectFifos.
+    # Each ObjectFifo can have one producer and one or multiple consumer ObjectFifoHandles.
+    # These endpoints can be used by the Workers, the Runtime or to generate connecting ObjectFifos
+    # (with forward(), split(), or join()).
+    of_in = ObjectFifo(tile_ty, name="in")
+    of_out = ObjectFifo(tile_ty, name="out")
+
+    # Task for the core to perform
+    def core_fn(of_in, of_out):
+        elem_in = of_in.acquire(1)
+        elem_out = of_out.acquire(1)
+        for i in range_(num_elements):
+            elem_out[i] = elem_in[i]
+        of_in.release(1)
+        of_out.release(1)
+
+    # Create a worker to perform the task
+    # The Worker takes as input a task and the input arguments it should run it with.
```

Comment:
input could be confusing here since the arguments of a Worker are a task and then a list of arguments (inputs and outputs) for that task.

Reply:
Fixed with your wording